### PR TITLE
Prepare 1.2.1 release

### PR DESCRIPTION
**Pull request by swissspidy**
Also adds missing readme content for the previous 1.2.0 release.

See #5723

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -110,7 +110,17 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
-= 1.2 =
+= 1.2.1 =
+
+**Release Date:** December 17, 2020.
+
+* New feature: status check in the editor
```
Comment (dreamofabear) [inline readme.txt]: Nit: Calling this a feature may confuse users who might look for its UI. How about "Automated plugin compatibility status check."?

**Review by spacedmonkey — APPROVED** [review]
Looking good. Should any of the doc blocks in php be changed to 1.2.1 over 1.2?

File: readme.txt [hunk 2]
```diff
@@ -138,6 +148,14 @@ For the plugin's full changelog, please see [the Releases page on GitHub](https:
 
 == Upgrade Notice ==
 
+= 1.2.1 =
+
+This release includes various bug fixes and improvements.
```
Comment (dreamofabear) [inline readme.txt]: Nit: "Bug fixes and performance improvements."

File: readme.txt [hunk 3]
```diff
@@ -138,6 +148,14 @@ For the plugin's full changelog, please see [the Releases page on GitHub](https:
...
+
+= 1.2.0 =
+
+This release includes initial support for animations and an all-new pre-publish checklist.
```
Comment (dreamofabear) [inline readme.txt]: Nit: "Added basic animations and a pre-publish checklist."

**Review by dreamofabear — APPROVED** [review]

**Comment by spacedmonkey** [comment]
These two jump out - https://github.com/google/web-stories-wp/blob/ae1d29b3a4cae506176209b263a30367f12e0cee/includes/Database_Upgrader.php#L337-L372

https://github.com/google/web-stories-wp/blob/ae1d29b3a4cae506176209b263a30367f12e0cee/includes/Media.php#L401-L417

File: readme.txt (hunk 1 above)
Comment (swissspidy) [reply readme.txt]: It doesn't have much to do with plugin compatibility though.

Maybe `New feature: automated site environment compatibility check`?

**Pull request merged**